retrospective-update/hydrosos_maps.py

```
@@ -0,0 +1,270 @@
+                    flow_values = flow_data.sel(river_id=linknos).sum(dim="river_id", skipna=True).values.item()
+                    cutoff_vals = flow_thresh_by_hydrobas_ds.sel(month=month, hybas_id=hybas_id)["flow_cutoff"].values
+                    category = classify_flow(flow_values, cutoff_vals)
+
+                    results.append({
+                        'year': year,
+                        'month': month,
+                        'hybas_id': hybas_id,
+                        'flow': flow_values,
+                        'class': category
+                    })
+                except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)